src/utils/reducers/cartReducer.js

```diff
@@ -0,0 +1,84 @@
+export const ADD_ITEM = 'ADD_ITEM';
+export const SET_ITEM_QUANTITY = 'SET_ITEM_QUANTITY';
+export const REMOVE_ITEM = 'REMOVE_ITEM';
+export const EMPTY_CART = 'EMPTY_CART';
+
+export const cartInitialState = {
```

Comment:
## Only a suggestion.
I think you can remove the `cart` property and only leave the `items` and `count` properties, the reducer name is `cartReducer.js` so with that is understanding the purpose of it.
```jsx
export const cartInitialState = {
    items: [],
    count: 0,
};
```